spec/requests/vendor_api/reject_application_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Vendor API - POST /applications/:application_id/reject', type: :request do
+  include VendorApiSpecHelpers
+
+  describe 'successfully rejecting an application' do
+    it 'returns rejected application' do
+      application_choice = create(:application_choice)
+      request_body = {
+        "data": {
+          "reason": 'Does not meet minimum GCSE requirements',
+          "timestamp": '2019-03-01T15:33:49.216Z',
```

Comment:
The date is included in reject application because the json schema requires that if a rejection exist in the schema it must have a date. 
Should this date be generated in the service then? :)

Reply:
This can be addressed in 
https://trello.com/c/7ubeCZJL/217-add-audit-trail-for-all-updates